components/core/NewWebsitePrototypeHeader.js

```diff
@@ -174,8 +174,14 @@ const openNavLink = {
   display: `flex`,
 };
 
-export const NewWebsitePrototypeHeader = (props) => {
+const NewWebsitePrototypeHeader = (props) => {
   const [open, setOpen] = useState(false);
+  const handleOpen = () => {
```

Comment:
This will produce bugs, please read this:

https://codedaily.io/tutorials/72/Creating-a-Reusable-Window-Event-Listener-Hook-with-useEffect-and-useCallback

The solution you are looking for is:

```
useEffect(() => {
  window.addEventListener("mouseup", props.onEvent);

  return () => window.removeEventListener("mouseup", props.onEvent);
});
```

* When we bind global events, we need to make sure we clean them up.